addon/routes/-lazy-loader.js

```diff
@@ -10,11 +10,11 @@ export default Ember.Route.extend({
       return;
     }
     retried = !retried;
-    transition.abort();
     var loadPromise = transition.intent.url ?
       this.get('lazyLoader').loadBundleForUrl(transition.intent.url) :
       this.get('lazyLoader').loadBundleForRouteName(transition.intent.name);
     return loadPromise.then(()=>{
+      transition.abort();
       transition.retry();
```

Comment:
Correct, we need to abort and retry eventually, but we don't want to abort before loading the code. After this change, we abort *after* we load the code, giving Ember an opportunity to render the loading states.